client/asset/btc/rpcclient.go

```diff
@@ -768,8 +769,26 @@ func (wc *rpcClient) walletLock() error {
 }
 
 // sendToAddress sends the amount to the address. feeRate is in units of
-// sats/byte.
+// sats/byte. If there is not a fee rate positional param, it is used
+// legacySendToAddress instead.
```

Comment:
This comment does not look correct. If wc.rpcCore.legacySentToAddr is true legacySendToAddress is used instead?

Reply:
Had typos in my suggestion from the other day, sorry. `legacySendToAddress` indeed looks most appropirate